worldline/README.md

```diff
@@ -0,0 +1,14 @@
+Worldline - e-Payment Company
+===
+
+Homepage:
+* https://w3id.org/worldline
+
+Lending Core Vocabulary
```

Comment:
Looks like you're not using this anymore?

Reply:
Why? In fact I need to create a permanent link from https://w3id.org/worldlines to https://github.com/worldlines/vocabulario
From there we are going to publish diferent kinds of vocabularies like lending-rdf.owl.

But I am a bit lost on how to do it.

Reply:
Looks like now it's just redirecting https://w3id.org/worldlines at the moment.  That's why I mentioned the "lending" part wasn't used.  We don't have a solid canonical example of how to do things, but there are plenty of other htaccess files here to look at.  Depending on what you need to serve up, you can add individual redirects or wildcards.  If you need help, list what kind of redirects you want and we can try and help.